File: app/src/main/java/com/tapan/weather/forecast/home/callback/HomeListener.java
package com.tapan.weather.forecast.home.callback;

import com.tapan.weather.forecast.core.base.callback.BaseListener;
import com.tapan.weather.forecast.home.view.HomeView;

public interface HomeListener<V extends HomeView> extends BaseListener<V> {

  void getWeatherContent(double latitude, double longitude);
}

File: app/src/main/java/com/tapan/weather/forecast/home/callback/HomeListener.kt
package com.tapan.weather.forecast.home.callback

import com.tapan.weather.forecast.core.base.callback.BaseListener
import com.tapan.weather.forecast.home.view.HomeView

interface HomeListener<V : HomeView> : BaseListener<V> {

  fun getWeatherContent(
    latitude: Double,
    longitude: Double
  )
}